content/pt/blue_green_deployment.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Blue Green Deployment
+status: Completed
+category: concept
+---
+
+## O que é
+
+O Blue-green deployment é uma estratégia para atualizar os sistemas de computador em execução com o mínimo de tempo de inatividade. O operador mantém dois ambientes, chamados "blue" e "green".
+Um atende ao tráfego de produção (a versão que os usuários estão usando atualmente), enquanto o outro é atualizado. Depois que o teste é concluído no ambiente inativo (green), o tráfego de produção é alternado (geralmente com o uso de um baleanceador de carga - load balancer). Observe que blue-green deployment geralmente significa alternar os ambientes por completo, abrangendo muitos serviços, todos de uma vez. Surpreendentemente, às vezes o termo é usado em relação a serviços individuais em um sistema. Para evitar essa ambiguidade, o termo "zero-downtime deployment" é preferido quando se refere a componentes individuais.
+
+## Qual problema é resolvido
+
+Blue-green deployments permitem tempo mínimo de inatividade ao atualizar o software que deve ser alterado em "lockstep" devido à falta de compatibilidade com versões anteriores. Por exemplo, blue-green deployment poderia ser apropriado para uma loja online consistindo de um site e um banco de dados que precisa ser atualizado, mas a nova versão do banco de dados não funciona com a versão antiga do site, e vice-versa. Neste caso, ambos precisam ser alterados ao mesmo tempo. Se isso fosse feito em um sistema de produção, os clientes perceberiam um tempo de inatividade.
```

Comment:
**Sugestion**: Blue-green deployments permitem tempo mínimo de inatividade ao atualizar o software que deve ser alterado em "lockstep" devido à falta de compatibilidade com versões anteriores. Por exemplo, blue-green deployment poderia ser apropriado para uma loja online consistindo de um site e um banco de dados que precisam ser atualizados, mas a nova versão do banco de dados não funciona com a versão antiga do site, e vice-versa. Neste caso, ambos precisam ser alterados ao mesmo tempo. Se isso fosse feito em um sistema de produção, os clientes perceberiam um tempo de indisponibilidade.

**Why:** typo fix (two services being updated, database and system - plural sentence)